### ISSUE TITLE
HUD only shows up for GM

### ISSUE THREAD

**Issue by handlepan12** [Larkinabout/fvtt-token-action-hud-pf2e#39]
The action-hud-pf2e works for me (DM) but my players are unable to use it. It does not show when they click their owned tokens no matter how I mess with the settings. 

The regular action hud (not made for pf2e) works fine but I'd prefer to use this one because its clearer to read.

**Comment by Larkinabout** [comment]
Is it possible they have toggle the HUD off using the 'Toggle HUD' hotkey (H by default)? There is also a module setting called 'Allow HUD', which needs to be set to 'Player'. Failing this, please ask the players (or log in yourself) to enable debugging within the module settings and output any logs referring to Token Action HUD in the console (press F12).

**Comment by jkwitchel** [comment]
I am having the same issue, had to install the original version.

**Comment by Larkinabout** [comment]
Can you trigger the toggle HUD via `game.tokenActionHud.toggleHud()`? You can execute this from within a macro.

**Comment by Larkinabout** [comment]
Can you try uninstalling both modules and reinstalling to see if that resolves the issue. I have tested it as working for both GM and players on my end. Failing that, if you're able to share a login temporarily, I can hop on and have a look at what's happening on the player side. You can message me on Discord at Russell#7071.

**Comment by MYFATSAUSSAGE** [comment]
I have the same issue. Flushing a user and recreating a new one seemed to work.

**Comment by Larkinabout** [comment]
Looks like the migration didn't quite work for the non-GM users as the data stored in the user flags should have been moved over to the data folder and removed from the user flag.

Glad you found a solution.

**Comment by Larkinabout** [comment]
Until I manage to put out a release with a fix. If anyone has the same issue and is OK with running scripts, you can remove the old flag data from the users and actors using this code:
```js
const actors = game.actors.filter(actor => actor.getFlag('token-action-hud-core', 'categories'))
for (const actor of actors) {
    actor.unsetFlag('token-action-hud-core', 'categories')
}

const users = game.users.filter(user => user.getFlag('token-action-hud-core', 'categories'))
for (const user of users) {
    user.unsetFlag('token-action-hud-core', 'categories')
    user.unsetFlag('token-action-hud-core', 'default')
}
```

**Comment by tianevitt** [comment]
Thanks for providing this workaround. It worked perfectly.

**Comment by Larkinabout** [comment]
Fixed in Token Action HUD Core 1.4.3.
